Bump pyvlx to 0.2.20

pergolafabio opened the pull request:
## Breaking change

## Proposed change
Update pyvlx to 0.2.20, see here for rls notes : https://github.com/Julius2342/pyvlx/releases/tag/0.2.20
Used by Velux : https://www.home-assistant.io/integrations/velux/

## Type of change
- [x] Dependency upgrade
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New integration (thank you!)
- [ ] New feature (which adds functionality to an existing integration)
- [ ] Breaking change (fix/feature causing existing functionality to break)
- [ ] Code quality improvements to existing code or addition of tests

## Additional information

- This PR fixes or closes issue: fixes #
- This PR is related to issue: 
- Link to documentation pull request: 

## Checklist

- [x] The code change is tested and works locally.
- [x] Local tests pass. **Your PR cannot be merged unless tests pass**
- [ ] There is no commented out code in this PR.
- [ ] I have followed the [development checklist][dev-checklist]
- [ ] The code has been formatted using Black (`black --fast homeassistant tests`)
- [ ] Tests have been added to verify that the new code works.

If user exposed functionality or configuration variables are added/changed:

- [ ] Documentation added/updated for [www.home-assistant.io][docs-repository]

If the code communicates with devices, web services, or third-party tools:

- [x] The [manifest file][manifest-docs] has all fields filled out correctly.  
      Updated and included derived files by running: `python3 -m script.hassfest`.
- [ ] New or updated dependencies have been added to `requirements_all.txt`.  
      Updated by running `python3 -m script.gen_requirements_all`.
- [x] For the updated dependencies - a link to the changelog, or at minimum a diff between library versions is added to the PR description. 
- [ ] Untested files have been added to `.coveragerc`.

The integration reached or maintains the following [Integration Quality Scale][quality-scale]:


- [x] No score or internal
- [ ] 🥈 Silver
- [ ] 🥇 Gold
- [ ] 🏆 Platinum

pergolafabio committed:
Bump pyvlx to 0.2.20

frenck reviewed (changes requested):
Thanks for bumping the dependency! However, could you please update the PR description to contain at least one (or multiple) of the following:

- A link to the release notes of this package version, and all versions in between.
- A link to the changelog of this package.
- A link to a Git(Hub) diff/compare view from the current version to the bumped version.

This allows us to review upstream changes, which is needed to decide this change is working as intended and/or if we can include it in, for example, a patch release of Home Assistant.

Additionally: None of the items in the checklist are checked, weren't you able to test it?

Thanks already! 👍

pergolafabio commented:
done, added link to releases, also tested indeed locally

frenck commented:
> done, added link to releases, also tested indeed locally

And non of the other checkboxes applied?

Please don't get me wrong, but we would appreciate if you took the time to submit a PR correct and fill out the PR template. That safes us all time :)

pergolafabio commented:
> > done, added link to releases, also tested indeed locally
> 
> And non of the other checkboxes applied?
> 
> Please don't get me wrong, but we would appreciate if you took the time to submit a PR correct and fill out the PR template. That safes us all time :)

Hi, sorry for trouble, updated the template now with links and checkboxes

frenck commented:
So, this one:

![image](https://user-images.githubusercontent.com/195327/149086702-a618361b-71f4-4d5d-81d4-66edeadb3ccd.png)

As this PR has updated a dependency, the task needs to be done and completed in order to get this PR built. It should not have passed the `pre-commit` task, yet you have checked:

![image](https://user-images.githubusercontent.com/195327/149086900-2bbaf0e1-ca5a-4f75-9c3b-090ca9cd811e.png)

Did you actually test this?

pergolafabio commented:
i load the velux component as a custom component, updated manifest with new version 0.2.20 and included version
all OK here

frenck commented:
> i load the velux component as a custom component, updated manifest with new version 0.2.20 and included version all OK here

That is not enough, as you can see. I'll trigger the CI builds, and it will show you it fails.

Please set up a development environment and run run `python3 -m script.gen_requirements_all` from the project root, as this PR is incomplete.

pergolafabio commented:
Sorry, no dev environment here... Will close this pr then
